src/views/RichTextView.tsx

```diff
@@ -19,6 +19,21 @@ type Props = {
     customReplacer?: Replacer,
 };
 
+function processImgSrcSet(srcset: string, meta: MetaData): string {
+    return srcset.split(/, */g).map(src => {
+        const srcParts = src.trim().split(' ');
+        switch (srcParts.length) {
+            case 1:
+                return getUrl(src, meta);
+            case 2:
+                return getUrl(srcParts[0], meta) + ' ' + srcParts[1];
+            default:
+                console.warn('Can not process image srcset: ' + src);
+                return src;
+        }
+    }).join(', ');
+}
```

Comment:
Move this to the url processor?
And export this, I would like to use it for other `<img>` tags also 😄